src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IFieldReferenceExpression.cs

```diff
@@ -241,5 +241,69 @@ void M()
 
             VerifyOperationTreeAndDiagnosticsForTest<MemberAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void IFieldReference_StaticFieldWithInstanceReceiver()
```

Comment:
>IFieldReference_StaticFieldWithInstanceReceiver [](start = 20, length = 47)

Please make sure we have tests for accessing a field (static and instance) of off a type

Reply:
Done.

---
In reply to: [149248465](https://github.com/dotnet/roslyn/pull/22972#discussion_r149248465) [](ancestors = 149248465)